docs/source/support/release-notes.rst

```diff
@@ -28,7 +28,7 @@ DAML Compiler
 DAML Standard Library
 ~~~~~~~~~~~~~~~~~~~~~
 
-- `parseInt` and `parseDecimal` now work at more extremes of values.
+- ``parseInt`` and ``parseDecimal`` now work at more extremes of values and accept leading plus signs.
```

Comment:
What exactly does “work at more extremes of values” mean?

Reply:
That is coming from @neil-da. We definitely couldn't parse the smallest `Int64` before. I don't know if there's more new stuff.